src/sshfs_mount/sftp_server.cpp

```diff
@@ -37,6 +37,9 @@ namespace mpl = multipass::logging;
 namespace
 {
 constexpr auto category = "sftp server";
+constexpr auto multipass_sshfs_bin_path = "/snap/multipass-sshfs/current/bin";
+constexpr auto multipass_sshfs_lib_path = "/snap/multipass-sshfs/current/lib";
```

Comment:
It irks me that this is hardcoded. Wonder if `snap run --shell multipass-sshfs.sshfs` would allow us to keep this dynamic?

`/snap` is unfortunately not uniform across distros.

Reply:
Ah, ok, didn't realize that `/snap` is not consistent.  Sure, I'll find a more general solution such as what you suggest.

Reply:
I had an idea: just have an `env` app inside `multipass-sshfs`, and `snap run multipass-sshfs.env` to get the right environment from the snap?

Reply:
These are gone now :grin: